src/qibocal/tests/test_niGSC_fitting.py

```diff
@@ -80,12 +78,21 @@ def test_exp2_fitting():
         # Distort ``y`` a bit.
         y_dist = y + np.random.uniform(-1, 1, size=len(y)) * 0.001
         popt, perr = fitting.fit_exp2_func(x, y_dist)
-        success += np.all(
+        worked = np.all(
             np.logical_or(
                 np.allclose(np.array(popt), [A2, A1, f2, f1], atol=0.05, rtol=0.1),
                 np.allclose(np.array(popt), [A1, A2, f1, f2], atol=0.05, rtol=0.1),
             )
         )
+        if not worked:
+            np.allclose(
+                popt[0] * popt[2] ** x + popt[1] * popt[3] ** x,
+                y_dist,
+                atol=0.01,
+                rtol=0.01,
+            )
+            worked = True
+        success += worked
```

Comment:
Stupid remark: if this is a counter, and not a boolean, I would reflect in the name
```suggestion
        successes += worked
```